MigrationPreparer/MigrationPreparer.json

```diff
@@ -0,0 +1,335 @@
+{
+    "$schema": "https://raw.githubusercontent.com/rdkcentral/rdkservices/main/Tools/json_generator/schemas/interface.schema.json",
+    "jsonrpc": "2.0",
+    "info": {
+        "title": "MigrationPreparer API",
+        "class": "MigrationPreparer",
+        "description": "The `MigrationPreparer` that is responsible for persisting and notifying listeners of any change of key/value pairs and get/set/reset value of MigrationReady RFC value"
```

Comment:
Migration preparer does not offer any notifications. Please correct the documentation (and also remove any code added for notification, if any)

Reply:
will update MigrationPreparer.json file.